python/ray/tests/test_runtime_env_env_vars.py

```diff
@@ -300,6 +300,25 @@ def get_options(val):
     assert pid7 == pid1
 
 
+def test_replaceable_environ(ray_start_regular):
+    @ray.remote
+    def get_env(key):
+        return os.environ.get(key)
+
+    assert (
+        ray.get(
+            get_env.options(
+                runtime_env={
+                    "env_vars": {
+                        "PATH": "${PATH}:/usr/local/bin",
```

Comment:
Windows tests are failing for `test_runtime_env_env_vars`, I think it's because on Windows PATH elements are separated by semicolon (`;`) instead of colon (`:`).

Reply:
Thanks, i'll fix.